parser/src/parser.cpp

```diff
@@ -204,16 +213,32 @@ int main(int argc, char* argv[])
     return 1;
   }
 
-  //--- Check workspace and project directory ---//
+  //--- Check database and project directory existence ---//
+
+  bool isNewDb = cc::util::connectDatabase(
+    vm["database"].as<std::string>(), false) ? false : true;
+  bool isNewProject = !checkProjectDir(vm);
+
+  if (isNewProject && !isNewDb || !isNewProject && isNewDb)
+  {
+    LOG(error)
+      << "Database and working directory existence are inconsistent. Use -f for reparsing!";
+    return 1;
+  }
+
+  if(!isNewDb)
```

Comment:
Space missing between if and condition.

Reply:
Solved in 4b52c300da8e3f8309356cf71e5f9c70207cc051